src/magql/resolver_factory.py

```diff
@@ -151,7 +151,24 @@ def resolve(self, parent, info, *args, **kwargs):
         return value
 
 
-class CamelResolver(Resolver):
+class ResultResolver:
+    """
+    Result Resolver retrieves the result key off of the payload object
+    :param parent: GraphQL Payload object
+    :param info: gql info dictionary
+    :return: The results field on the payload object
+    """
+
+    def __call__(self, parent, info, *args, **kwargs):
+        value = (
+            parent.get("result")
+            if isinstance(parent, dict)
+            else getattr(parent, "result", None)
+        )
+        return value
```

Comment:
I changed this to `return parent` without changing anything else, and the tests still passed. Needs a failing test.